src/shared/coredump-util.c

```diff
@@ -158,7 +158,7 @@ int parse_auxv(int log_level,
 }
 
 int set_coredump_filter(uint64_t value) {
-        char t[STRLEN("0xFFFFFFFF")];
+        char t[DECIMAL_STR_MAX(uint64_t)]; /* For the same number decimal >> hex */
```

Comment:
hmm, i find this icky. if you did the same for uint8_t, then DECIMAL_STR_MAX(uint8_t) would return 4 (i.e. `sizeof("255")`). But actually you need 5 bytes space, i.e. `sizeof("0xFF")`...

Hence, maybe just do it right:

```c
/* Maximum buffer size needed for formatting an unsigned integer type as hex, including space for `0x` prefix and trailing NUL suffix */
#define HEXADECIMAL_STR_MAX(type) (2+sizeof(type)*2+1)
```

It's short, then requires no further comment, and is correct for any unsigned integer type. yay!